drivers/windows/file_access_windows.cpp

```diff
@@ -127,7 +127,7 @@ Error FileAccessWindows::open_internal(const String &p_path, int p_mode_flags) {
 	}
 
 #ifdef TOOLS_ENABLED
-	// Windows is case insensitive, but all other platforms are sensitive to it
+	// Windows is case insensitive, but other platforms can be sensitive to it
```

Comment:
I would just remove this sentence, or at least add a "by default" for Windows, because its also not strictly true anymore: https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity

Reply:
Update to:

```c++
// Windows is case insensitive in the default configuration, but other platforms can be sensitive to it
```